### PR TITLE
Added "christmasify" avatar effect command.

### LINKED ISSUE #937: New ".christmasify" command
## Description
Changes a users current profile picture into something more seasonal, similar to commands like .spookify

## Reasoning
It would be fun and spread the holiday cheer

## Proposed Implementation
Adding Christmas hats, snowflakes, etc etc 

## Would you like to implement this yourself?
- [ ] I'd like to implement this feature myself
- [x] Anyone can implement this feature

I would do it myself, but I'm rather busy and don't know how to use pillow, as I've never experimented with it before

### PR DESCRIPTION
**Pull request by SockYeh**
## Relevant Issues

Closes #937 

## Description

* Added "christmasify" command to add a Christmas touch to user's avatar.
* Currently has 2 effects - Room for future additions
    * Make avatar chilly
    * Add ornaments
## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/avatar_modification/_effects.py [hunk 1]
```diff
@@ -294,3 +294,26 @@ def mosaic_effect(image: Image.Image, squares: int) -> Image.Image:
...
+        """Applies the christmas effect to the given image."""
+        christmas_effects = ["ice", "ornaments"]
+        overlay_type = random.choice(christmas_effects)
+        overlay = Image.open(Path(f"bot/resources/holidays/christmas/{overlay_type}_frame.png"))
+
+
+        if overlay_type == "ice":
+            overlay = overlay.resize((1024, 1024))
+
+            image = ImageOps.colorize(image.convert("L"), (0, 0, 0), (212, 235, 242))
```
Comment (wookie184) [inline bot/exts/avatar_modification/_effects.py:309]: What is this doing?

**Review by wookie184 — CHANGES_REQUESTED** [review]
This looks great :)

Where did you get `ice_frame.png` and `ornaments_frame.png` from? We need to make sure they're licensed appropriately so we can use them here. I also wonder if there's a higher quality version of `ice_frame.png`, currently it looks quite blurry.

File: bot/exts/avatar_modification/_effects.py (hunk 1 above)
Comment (SockYeh) [reply bot/exts/avatar_modification/_effects.py:309]: it adds a blue hue to the img

**Comment by SockYeh** [comment]
> This looks great :)
> 
> Where did you get `ice_frame.png` and `ornaments_frame.png` from? We need to make sure they're licensed appropriately so we can use them here. I also wonder if there's a higher quality version of `ice_frame.png`, currently it looks quite blurry.

It was just downloaded from google so i have no clue where the source actually is 😅

**Comment by SockYeh** [comment]
> > > This looks great :)
> > > Where did you get `ice_frame.png` and `ornaments_frame.png` from? We need to make sure they're licensed appropriately so we can use them here. I also wonder if there's a higher quality version of `ice_frame.png`, currently it looks quite blurry.
> > 
> > 
> > It was just downloaded from google so i have no clue where the source actually is 😅
> 
> Reverse image search for `ornaments_frame.png` led me to [this](https://www.hiclipart.com/free-transparent-background-png-clipart-plegr) (among other similar looking websites). They've mentioned that they do not require attribution for non-commercial uses.
> 
> `ice_frame.png` can be found on freepik.com [here](https://www.freepik.com/free-vector/blue-snowflakes-frame_829245.htm) and they've stated that they require attribution. (Also I do not see a `.png` download option but they do have `.svg`). It seems that the same author has also uploaded the image to [shutterstock](https://www.shutterstock.com/image-vector/snowflake-frame-340281305).

that's great! What is the next step?

**Comment by wookie184** [comment]
Ideally we'd use assets that don't require attribution, since that's the easiest option where possible.

For ones that do require attribution, I think it's OK to put it in the command's help text.

**Comment by SockYeh** [comment]
I am unable to find images so would be grateful if someone could find one to add to the bot. Thanks.

**Comment by ChrisLovering** [comment]
Once copyright free images have been found, we can look into opening this back up and implementing this change.

**Pull request closed without merging**